docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
```

Comment:
The `build` context is set to the current directory. Ensure that your `Dockerfile` is correctly configured and located in the root of your project directory.